src/main/java/mate/academy/spring/controller/UserController.java

```diff
@@ -24,7 +25,7 @@ public UserController(UserService userService,
     @GetMapping("/by-email")
     public UserResponseDto findByEmail(@RequestParam String email) {
         User user = userService.findByEmail(email).orElseThrow(
-                () -> new RuntimeException("Invalid email"));
+                () -> new DataProcessingException("Invalid email"));
```

Comment:
return the original exception, DataProcessingException was created only for dao layer